Process.h

```diff
@@ -174,8 +174,8 @@ typedef struct Process_ {
    /*
     * Internal time counts for showing new and exited processes.
     */
-   time_t seenTs;
-   time_t tombTs;
+   uint64_t seenMs;
+   uint64_t tombMs;
```

Comment:
Why rename these fields?

Reply:
> [...] let's change it back if its confusing.

@smalinux are you happy to do the rename?  I'm about to go on holidays for a few days.  :)

Reply:
@BenBE oh, also, time_t is typically a time-in-seconds (since the epoch) - hence the change in type (i.e. that part should stay as-is IMO).

Reply:
> @smalinux are you happy to do the rename?

@natoscott Yes, certainly, happy holiday :)

Reply:
I'm both fine with the type change (that should stay) and also with the `Ms` marker at the end. It's just a bit unfortunate to drop the mid-part `Ts` altogether. Maybe `seenStampMs`/`tombStampMs`?

Reply:
@BenBE @smalinux that sounds good to me